dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -0,0 +1,138 @@
+package com.microsoft.reactnativedualscreen.dualscreen
+
+import android.content.Context
+import android.graphics.Rect
+import android.view.View
+import android.view.WindowManager
+import com.facebook.react.bridge.*
+import com.facebook.react.bridge.Arguments.createMap
+import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter
+import com.microsoft.device.display.DisplayMask
+
+const val HINGE_WIDTH_KEY = "hingeWidth"
+const val IS_DUALSCREEN_DEVICE_KEY = "isDualScreenDevice"
+const val FEATURE_NAME = "com.microsoft.device.display.displaymask"
+
+class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+    private val isDualScreenDevice: Boolean
+        get() =  reactApplicationContext.packageManager.hasSystemFeature(FEATURE_NAME)
+	private val mDisplayMask: DisplayMask
+		get() = DisplayMask.fromResourcesRectApproximation(currentActivity)
```

Comment:
>currentActivity [](start = 53, length = 15)

I'm hearing on Discord that the currentActivity can be null in some situations (search for posts from kmelmon on android channel).  How do we correctly handle this?

Reply:
fixed in PR #3